src/components/design_system/modal/Modal.module.css

```diff
@@ -1,2 +1,74 @@
-/* PLACEHOLDER FOR DESIGN SYSTEMS COMPONENT CSS
-see notes in component file */
+.editForm {
+  width: 100%;
+}
+
+.modalContent {
```

Comment:
Is .modalContent used later on? If not, that could be a contender for code cleanup.

Reply:
It's used as styling for the other modals in the file (not the edit student one) so I left it in there, but we could maybe change the classnames to make the difference more clear?